.github/workflows/test.yaml

```diff
@@ -58,6 +58,11 @@ jobs:
         run: |
           source $VENV
           pytest
+      - name: Test Database Backends
+        if: ${{ !inputs.coverage }}
```

Comment:
Do we not want coverage for those? If not, I think we need to exclude that in the config as well, since otherwise it will be counted as missing.

Reply:
Should I remove the whole entry or just the `if` clause?

Reply:
Hmm. I think ideally all tests in the CI would include the database tests. You should be able to override the ini "adopts" option somewhere but I'm not 100% sure what the config for that would look.

Reply:
I think the updated configuration accomplishes all of the above.